src/components/common/Extensions/ImageCarousel/ImageCarousel.tsx

```diff
@@ -11,10 +11,18 @@ const displayName = 'common.Extensions.ImageCarousel';
 const ImageCarousel: FC<ImageCarouselProps> = ({
   slideUrls = images,
   options = { loop: true, align: 'start' },
+  isAutoplay = false,
+  isImageFullWidth,
```

Comment:
what if we rework `slideUrls` to be an array of image urls and classNames? would be a bit cleaner, since now we control all images from one variable